Dockerfile

```diff
@@ -87,6 +87,8 @@ COPY workflows workflows
 COPY container-simple-checks.sh test_cmds.txt ./
 RUN chmod u+x /galaxy/ansible/run_galaxy_config.sh
 
+ENV GALAXY_CONFIG_BRAND PhenoMeNal
```

Comment:
Where is that ENV used ?

Reply:
Found it in https://github.com/bgruening/docker-galaxy-stable#galaxys-config-settings--toc